### PR TITLE
LPD-21965 | LPD-21966: Allow to export and import AutoIncrement Object Field

### PR DESCRIPTION
**Pull request by sergiojimcos**
## Motivation
The idea of the PR is to allow users to export and import AutoIncrement Object Fields.

**What is new?**
- Update `BatchEngineBrokerTest` to add new data for export and import Auto Increment Object Field
- Allow to export Auto Increment Object Field if the Feature Flag is enabled
- Update import Playwright tests and add test case for Unique Constraint (when two AutoIncrement Object Fields has the same value)

**Note:** In order to create AutoIncrement Object Fields, you need to set `feauture.flag.LPS-196724=true`

**How to reproduce**
1. Deploy `object-rest-impl`

**JIRA Related Tickets**
https://liferay.atlassian.net/browse/LPD-21966
https://liferay.atlassian.net/browse/LPD-21965

### CODE REVIEW

**Comment by sergiojimcos** [comment]
ci:test:relevant

**Comment by sergiojimcos** [comment]
ci:test:relevant

File: modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java [hunk 1]
```diff
@@ -1179,6 +1181,34 @@ private ObjectDefinition _publishObjectDefinition(
...
 						).build(),
+						new AutoIncrementObjectFieldBuilder(
+						).labelMap(
+							LocalizedMapUtil.getLocalizedMap(
+								RandomTestUtil.randomString())
+						).name(
+							"testAutoIncrementField"
+						).objectFieldSettings(
+							Arrays.asList(
+								new ObjectFieldSettingBuilder(
```
Comment (ccorreagg) [inline modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java:1192]: I see a pattern of using `_createObjectFieldSetting` instead

Comment (sergiojimcos) [reply modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java:1192]: I will update it, I just followed the Object Services pattern

**Comment by ccorreagg** [comment]
ci:test:sf

**Comment by ccorreagg** [comment]
ci:stop

**Comment by ccorreagg** [comment]
ci:forward

**Comment by ccorreagg** [comment]
ci:report:2558526519

**Comment by ccorreagg** [comment]
ci:forward

**Comment by sergiojimcos** [comment]
ci:report:2558615516

**Comment by sergiojimcos** [comment]
ci:forward:force

**Pull request closed without merging**